tests/utils-test.js

```diff
@@ -113,5 +113,11 @@ describe('utils', () => {
         utils.populateQuery({}, undefined)
       }).not.to.throw()
     })
+
+    it('does not throw error when query dependency is not present', function () {
```

Comment:
This looks troubling since it's this exception that determines if the `select` should be disabled or not. Perhaps I need to find a better way to determine this since `{}` is still a valid query.